app/controllers/notes_controller.rb

```diff
@@ -178,6 +178,7 @@ def update
       @revision = @node.latest
       @revision.title = params[:title]
       @revision.body = params[:body]
+      @revision.timestamp = Time.now.to_i
       if params[:tags]
         params[:tags]&.tr(' ', ',')&.split(',')&.each do |tagname|
```

Comment:
What difference does it make to convert ` ` to `,` and then split on `,` instead of directly splitting on ` `?

Reply:
it's not added by me

Reply:
Oops, sorry. I did not realise that.
@jywarren can have a look at this, and if I'm right, this can be a good FTO candidate.